{"sections":[],"schemaVersion":{"patch":0,"major":0,"minor":3},"kind":"symbol","abstract":[{"type":"text","text":"Inherited from "},{"type":"codeVoice","code":"ErrorAttribute.valueDescription"},{"type":"text","text":"."}],"hierarchy":{"paths":[["doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/TraceErrorAttribute"]]},"metadata":{"symbolKind":"property","externalID":"s:13SwiftErrorKit05TraceB9AttributeV16valueDescriptionSaySSGvp","modules":[{"name":"SwiftErrorKit"}],"roleHeading":"Instance Property","title":"valueDescription","role":"symbol","fragments":[{"kind":"keyword","text":"var"},{"text":" ","kind":"text"},{"text":"valueDescription","kind":"identifier"},{"kind":"text","text":": ["},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"kind":"text","text":"]"}]},"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/swifterrorkit\/traceerrorattribute\/valuedescription"]}],"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"valueDescription","kind":"identifier"},{"kind":"text","text":": ["},{"preciseIdentifier":"s:SS","kind":"typeIdentifier","text":"String"},{"kind":"text","text":"] { "},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}]}],"identifier":{"url":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/TraceErrorAttribute\/valueDescription","interfaceLanguage":"swift"},"references":{"doc://SwiftErrorKit/documentation/SwiftErrorKit/TraceErrorAttribute":{"type":"topic","fragments":[{"kind":"keyword","text":"struct"},{"kind":"text","text":" "},{"kind":"identifier","text":"TraceErrorAttribute"}],"abstract":[{"type":"text","text":"An error attribute to track a source location."}],"title":"TraceErrorAttribute","identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/TraceErrorAttribute","navigatorTitle":[{"kind":"identifier","text":"TraceErrorAttribute"}],"kind":"symbol","url":"\/documentation\/swifterrorkit\/traceerrorattribute","role":"symbol"},"doc://SwiftErrorKit/documentation/SwiftErrorKit/TraceErrorAttribute/valueDescription":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit\/TraceErrorAttribute\/valueDescription","url":"\/documentation\/swifterrorkit\/traceerrorattribute\/valuedescription","role":"symbol","abstract":[],"fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"text":"valueDescription","kind":"identifier"},{"kind":"text","text":": ["},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"kind":"text","text":"]"}],"title":"valueDescription","type":"topic","kind":"symbol"},"doc://SwiftErrorKit/documentation/SwiftErrorKit":{"identifier":"doc:\/\/SwiftErrorKit\/documentation\/SwiftErrorKit","abstract":[{"type":"text","text":"Error attribution and commons."}],"type":"topic","title":"SwiftErrorKit","kind":"symbol","role":"collection","url":"\/documentation\/swifterrorkit"}}}